_source/_posts/2022-07-29-python-okta.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: blog_post
+title: "Use the Okta CLI to Build Secure Python Apps"
+author: erikka-innes
+by: contractor
+communities: [python]
+description: "Learn how to set up the Okta CLI and run the hosted authentication sample application."
+tags: [python, flask, authentication]
+tweets:
+- "Configure the Okta CLI and run your first sample application using hosted authentication!"
+- "Try the Okta CLI, the easiest way to create your first Okta application with hosted authentication."
+image: blog/python-okta/social.jpg
+type: conversion
+---
+
+Okta is a customizable, secure solution that lets you add authentication and access management to your applications. Since this can be a complex topic, we recommend setting up your first project with our CLI. In this walkthrough, we will:
+- Configure the Okta CLI
+- Teach you how to create your first application
+- Learn about authentication and authorization credentials
+- Set up Hosted Authentication with Okta and try it out 
+- Cover troubleshooting issues you might encounter
+
+**Prerequisites**
+
+This tutorial uses the following technologies but doesn't require any prior experience:
+- Okta developer account - these instructions assume you already have signed up. If you didn't, you can sign up for an account using the CLI! 
```

Comment:
```suggestion
- Okta Developer account - these instructions assume you already have signed up. If you didn't, you can sign up for a free account using the CLI! 

Capitalizing this, since the Dev account is a unique thing, also letting readers know it's free.